### PR TITLE
Add title for input field and fix button #418

### PR DESCRIPTION
**Pull request by draganakovacevic**

### CODE REVIEW

**Comment by stanislavgrujic** [comment]
@aberonja @BraneOpacic21 Please review.

File: web/src/main/frontend/src/base/stores/AuthStore.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import {observable, action} from 'mobx';
 import authApi from '../rest/authApi';
-import {AUTH_TOKEN_KEY} from '../../components/utility/httpUtil';
+import {AUTH_TOKEN_KEY, USER_IS_ADMIN} from '../../components/utility/httpUtil';
 
 class AuthStore {
-  @observable isAdmin = true;
+
```
Comment (BraneOpacic21) [inline web/src/main/frontend/src/base/stores/AuthStore.js:6]: Same as in #421

File: web/src/main/frontend/src/base/stores/AuthStore.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+  @observable isAdmin;
```
Comment (BraneOpacic21) [inline web/src/main/frontend/src/base/stores/AuthStore.js:8]: Same as in #421

File: web/src/main/frontend/src/components/DashboardTableBody.js [hunk 3]
```diff
@@ -32,27 +32,41 @@ class DashboardTableBody extends React.Component {
...
-              />
-              <button
-                className="dashboard__trash_icon"
-                onClick={this.props.confirmPopup.showPopup(this.rowDelete(row))}
-                title="Delete test..."
-              />
-            </td>
-          </React.Fragment>
-        )}
+
```
Comment (BraneOpacic21) [inline web/src/main/frontend/src/components/DashboardTableBody.js:35]: Same as in #421

File: web/src/main/frontend/src/components/DashboardTableBody.js [hunk 4]
```diff
@@ -32,27 +32,41 @@ class DashboardTableBody extends React.Component {
...
+        <React.Fragment>
+          {this.props.isAdmin === 'true' ?
```
Comment (BraneOpacic21) [inline web/src/main/frontend/src/components/DashboardTableBody.js:37]: Same as in #421

File: web/src/main/frontend/src/scss/components/_reviewTest.scss [hunk 5]
```diff
@@ -140,6 +140,7 @@ textarea {
   overflow: hidden !important;
 
   .CodeMirror-scroll {
+    z-index: 0;
```
Comment (BraneOpacic21) [inline web/src/main/frontend/src/scss/components/_reviewTest.scss:143]: Same as in #421

**Pull request merged**